themes/nord-color-theme.json

```
@@ -1215,6 +1215,45 @@
         "foreground": "#81A1C1"
       }
     },
+    {
+      "name": "[Rust] Entity types",
+      "scope": [
+        "source.rust entity.name.type"
+      ],
+      "settings": {
+        "foreground": "#8FBCBB"
+      }
+    },
+    {
+      "name": "[Rust] Macro",
+      "scope": [
+        "meta.macro.rust"
+      ],
```

```suggestion
      "scope": "source.rust meta.macro entity.name.function",
```

Using `source.rust` makes it more explicit as well as using narrow it down to `entity.name.function` so that it only affects functions and no other tokens that match the `meta.macro` group.